File: python/exercício_36.py
print()
print("=-"*60)
print()
print("--------------Notas das provas------------------")
print()
print("=-"*60)
print() 
n1=float(input("Digite a nota da primeira prova: "))
n2=float(input("Digite a nota da segunda prova: "))
n3=float(input("Digite a nota da terceira prova: "))
n4=float(input("Digite a nota da quarta prova: "))
n5=float(input("Digite a nota da quinta prova: "))
media = 70

if n1 >= media:
    if n2 >= media:
        if n3 >= media:
            if n4 >= media:
                if n5 >= media: 
                    print("A – passou em todos os exames!")
if n1 >= media:
    if n2 >= media:
        if n3 < media:
            if n4 >= media:
                if n5 < media:
                    print("B – passou em I, II e IV, mas não em III ou V.")
if n1 >= media:
    if n2 >= media:
        if n3 >= media:
            if n4 >= media:
                if n5 < media: 
                    print("C – passou em I e II, III ou IV, mas não em V.") 
else: 
    print("Reprovado - outra situação")        

File: java/exercício_36.java
import java.util.Scanner;

public class exercício_36 {
    public static void main(String[] args) {

        try (Scanner nota = new Scanner(System.in)){

        System.out.printf("Digite a nota da primeira prova: ");
        Float nota1 = nota.nextFloat();
        System.out.printf("Digite a nota da segunda nota: ");
        float nota2 = nota.nextFloat();
        System.out.printf("Digite a nota da Terceira prova: ");
        float nota3 = nota.nextFloat();
        System.out.printf("Digite a nota da Quarta prova: ");
        float nota4 = nota.nextFloat();
        System.out.printf("Digite a nora da Ultima prova: ");
        float nota5 = nota.nextFloat();

        int media = 70;

        if (nota1 >= media) {
            if (nota2 >= media)
                if (nota3 >= media)
                    if (nota4 >= media)
                        if (nota5 >= media)
                            System.out.printf("Voce passou em todas as provas!");
        }

        if (nota1 >= media) {
            if (nota2 >= media)
                if (nota3 < media)
                    if (nota4 >= media)
                        if (nota5 < media)
                            System.out.printf("B – passou em I, II e IV, mas não em III ou V.");
        }

        if (nota1 >= media) {
            if (nota2 >= media)
                if (nota3 >= media)
                    if (nota4 >= media)
                        if (nota5 < media)
                            System.out.printf("C – passou em I e II, III ou IV, mas não em V.");
        } else {
            System.out.printf("Reprovado - outra situação");
        }
    }
}
}
